### DISCUSSION TITLE
Mailserver Migration

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#3379] [Q&A] [answered]
I have the mailserver up and running well incl 1 domain and 2 users' emails.
Now I need to migrate the mailserver to another server. I can pull the docker image from there but how do I integrate the existing mail accounts? Any migration tools? Ideas which folders to copy over?

**Comment by casperklein** [comment, marked as answer]
If you use the default `compose.yaml` file, all you have to do is to copy over the `docker-data` directory, which contains all necessary files:

https://github.com/docker-mailserver/docker-mailserver/blob/86e18d04ddf6b39f33a36e2fd5d8b388265656de/compose.yaml#L18-L21

**Reply by MarcS1975** [reply]
Thanks for your quick reaponse. I will try this.
I guess mailserver uses an Sql database so I woukd have to stop the old server before copying files over. Correct?

**Reply by casperklein** [reply]
It's all file based. But it's best practice to stop the container before copy the data. Otherwise files could be change (for example log files) while copying.
